easybuild/easyconfigs/r/Rust/Rust-1.70.0-GCCcore-12.3.0.eb

```diff
@@ -13,8 +13,8 @@ source_urls = ['https://static.rust-lang.org/dist/']
 sources = ['rustc-%(version)s-src.tar.gz']
 patches = ['Rust-1.70_sysroot-fix-interpreter.patch']
 checksums = [
-    {'rustc-%(version)s-src.tar.gz': 'b2bfae000b7a5040e4ec4bbc50a09f21548190cb7570b0ed77358368413bd27c'},
-    {'Rust-1.70_sysroot-fix-interpreter.patch': '72bffa54837d1a2a117ad63c16deb4078a56a3d9d4f4cbc7430d68dd4cb3effc'},
+    {'rustc-1.70.0-src.tar.gz': 'b2bfae000b7a5040e4ec4bbc50a09f21548190cb7570b0ed77358368413bd27c'},
```

Comment:
`--inject-checksums` uses hardcoded version rather the `%(version)s`, there's little reason to fight that imho